chunky/src/java/se/llbit/chunky/entity/ArmorStand.java

```diff
@@ -620,4 +621,9 @@ public static Entity fromJson(JsonObject json) {
   @Override public JsonObject getGear() {
     return gear;
   }
+
+  @Override
+  public boolean shouldLoad() {
+    return PersistentSettings.getLoadArmorStands();
```

Comment:
I feel like this should not be a persistent setting but something saved in the scene. If I create a sceneA with entities, then create a sceneB without entities, then load sceneA, I want my entities to load.
Like other settings saved in the scene, the default value could still be from the persistent settings in addition.
Maybe this means you need to change the `shouldLoad` interface to take something as parameter (the whole feels overkill but is maybe the simplest if you just add some booleans to the scene, maybe do something a bit better organized and put those boolean in some kind of `EntityLoadingOption` class or I don't know)

Reply:
Aren't entities stored in the scene anyway? These checkboxes only affect the loading of entities when loading chunks, but not when loading a scene from disk.

Reply:
Yes they are but maybe I want to reload the chunks after reloading sceneA because I want to use emitter sampling or I changed something in my minecraft world or whatever. 
Maybe let's wait to see wait @leMaik thinks

Reply:
Just saw this now (after rebasing the PR)… I agree with @aTom3333 on this one (reloading the chunks of an existing scene shouldn't cause entities to disappear). 
Saving this in the scene also has the nice benefit that the scene will still work as expected when the octree is removed and it is opened.